ProcessMaker/modeler
# FOUR-11840 | Add PM Blocks to the Shortcut Bar

# Feature
Ticket: [FOUR-11840](https://processmaker.atlassian.net/browse/FOUR-11840)

This enhancement to the Process Modeler allows Designers to pin any PM Block to the Shortcut Bar. This feature simplifies the workflow for Designers seeking quick access to frequently used PM Blocks.

# How to Test
1. Go to branch `feature/FOUR-11840` in `modeler` and branch `feature/FOUR-11839` in `package-pm-blocks`
2. Have PM Blocks created.
3. Open a Process.
4. Navigate to the PM Blocks tab in the left rail of the Modeler.
5. Pin a PM Block.
	- The PM Block should appear pinned in the PM Blocks tab.
	- The PM Block should appear pinned in the 'Add Object' tab.
	- The PM Block should appear in the Shortcut Bar at the bottom of the Modeler.
6. Pin another object (not a PM Block).
	- The object should not appear pinned in the PM Blocks tab.

ci:package-pm-blocks:feature/FOUR-11839
ci:next
ci:deploy

# Code Review Checklist



[FOUR-11840]: https://processmaker.atlassian.net/browse/FOUR-11840?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
